src/String/String.res

```diff
@@ -16,6 +16,10 @@ export concat = (str, append) => Js.String2.concat(str, append)
 )
 export head = str => Js.String2.slice(str, ~from=0, ~to_=1)
 
+%comment(
+  "Returns `Some(value)` where `value` is the last character of the string, or `None` if the given string is empty."
+)
+
```

Comment:
seems like this comment should be moved to the line before the `last` function :)

Reply:
oh you are right!